hedera-node/hedera-app/src/main/java/com/hedera/node/app/records/impl/producers/BlockStreamFormat.java

```diff
@@ -38,13 +39,37 @@
  */
 public interface BlockStreamFormat {
 
+    /**
+     * Given the starting running hash and a stream of serialized {@link BlockItem}s, compute the new running hash by
+     * adding each {@link BlockItem}'s hash to the running hash one at a time.
```

Comment:
I'm still unsure if we're keeping this running hash or not. I had the impression the running hash is going away, and instead we'll use the hash inside of each block's `BlockProof`. But, if that is the case, are we maintaining evm equivalence only by keeping the hashes of the last 256 blocks? Or, do we need a per-transaction running hash to report in addition to the running block hash?